packages/components/src/components/dropdown/display-text.jsx

```diff
@@ -56,7 +56,7 @@ DisplayText.propTypes = {
     list: listPropType(),
     name: PropTypes.string,
     placeholder: PropTypes.string,
-    value: PropTypes.string,
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
In some components like `dropdown` the value is a number.

Reply:
@hamid-deriv Can’t we keep it as `string` and instead cast the number to string in those places to have better type safety? 🤔 

Although it seems to be outside of your PR scope, Please ignore this comment 🙇🏻